### PR TITLE
feat: make image zoomable by adding package

### LINKED ISSUE #912: Make documentation images zoomable
# Motivation

As a user I would like to be able to click on an image to have it zoom in

# Details

Here's an example of it an action:
https://docusaurus.io/docs/next/advanced/architecture

Seems possible just by putting images in a React component:
https://github.com/facebook/docusaurus/issues/6713#issuecomment-1045511347

This looks to be the implementation:
https://github.com/facebook/docusaurus/blob/main/website/docs/advanced/architecture.mdx
(wrapping the markdown image in a `<Zoom>` component from `react-medium-image-zoom`)

### PR DESCRIPTION
**Pull request by kittybest**
# Description

as described in #912 .

## Additional Notes

Install a package for automatically make markdown images zoomable: https://www.npmjs.com/package/docusaurus-plugin-image-zoom

## Related issue(s)

close #912 .

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: website/docusaurus.config.ts [hunk 1]
```diff
@@ -172,6 +173,7 @@ const config: Config = {
       ],
       copyright: `Copyright © ${new Date().getFullYear()} Privacy and Scaling Explorations`,
     },
+    zoom: {},
```
Comment (0xmad) [inline website/docusaurus.config.ts:176]: Do we need to add selectors and other configuration here?

**Review by 0xmad — APPROVED** [review]
@kittybest thanks, just one question

File: website/docusaurus.config.ts (hunk 1 above)
Comment (kittybest) [reply website/docusaurus.config.ts:176]: yes, I tried to remove it and it will cause the package not working

Comment (0xmad) [reply website/docusaurus.config.ts:176]: Here I mean the zoom configuration like it described in package readme.

```
zoom: {
      selector: '.markdown :not(em) > img',
      background: {
        light: 'rgb(255, 255, 255)',
        dark: 'rgb(50, 50, 50)'
      },
      config: {
        // options you can specify via https://github.com/francoischalifour/medium-zoom#usage
      }
    }
```

Do we need to configure it somehow or it works fine without it?

Comment (kittybest) [reply website/docusaurus.config.ts:176]: Sorry for late replay
We don't need to add those config since they are default values

**Pull request merged**